app/src/test/java/edu/ucandroid/weathernotice/CountryUnitTest.kt

```diff
@@ -63,8 +57,8 @@ class ExampleUnitTest {
         println("")
     }
 
-    private fun thenResultsShouldContainBelize() {
-        var containsCincinnati:Boolean = false
+    private fun thenResultsShouldContainCincinnati() {
+        var containsCincinnati = false
```

Comment:
Again, simple refactoring. Also, removed the explicit declaration of the "containsCincinnati" boolean.

Citation: https://kotlinlang.org/docs/coding-conventions.html#formatting

Reply:
Good catch, thank you.